draft-ietf-gnap-core-protocol.md

```diff
@@ -420,6 +430,11 @@ that returns from the interaction.
     in the form of [access tokens](#response-token) and
     [direct subject information](#response-subject) to the client instance.
 
+10. The client instance [uses the access token](#use-access-token) to call the RS.
```

Comment:
For the sake of consistency, I would suggest using only "presents the access token" or "uses the access token" everywhere.

Reply:
Thank you, I actually saw this during editing but missed it here!